src/Parsers/tests/gtest_Parser.cpp

```diff
@@ -215,7 +215,7 @@ INSTANTIATE_TEST_SUITE_P(ParserCreateDatabaseQuery, ParserTest,
         },
         {
             "CREATE DATABASE db ENGINE=Foo TABLE OVERRIDE `tbl` (), TABLE OVERRIDE a (COLUMNS (_created DateTime MATERIALIZED now())), TABLE OVERRIDE b (PARTITION BY rand())",
-            "CREATE DATABASE db\nENGINE = Foo\nTABLE OVERRIDE `tbl`,\nTABLE OVERRIDE `a`\n(\n    COLUMNS\n    (\n        `_created` DateTime MATERIALIZED now()\n    )\n),\nTABLE OVERRIDE `b`\n(\n    PARTITION BY rand()\n)"
+            "CREATE DATABASE db\nENGINE = Foo\nTABLE OVERRIDE `tbl`\n(\n\n),\nTABLE OVERRIDE `a`\n(\n    COLUMNS\n    (\n        `_created` DateTime MATERIALIZED now()\n    )\n),\nTABLE OVERRIDE `b`\n(\n    PARTITION BY rand()\n)"
```

Comment:
btw what is the point in having an empty table override?

Reply:
Useless, right, but seems it was made to allow not only adding in CREATE, but also ALTER DATABASE ... MODIFY|DROP|ADD TABLE OVERRIDE, which is not yet available.  We already have implementation of this, and was wondering how to contribute: either fixing this bug firstly with not parsable empty overrides, or pushing all changes together with ALTER MODIFY|DROP|ADD support.